From e831e4cf00633500ae30b2fb301ae7df3e4e8ebc Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 31 May 2020 11:02:42 +0200
Subject: [PATCH 1/1] virtio: VIRTIO_RNG depends on DM_RNG

Add the missing Kconfig dependency and let VIRTIO_RNG default to yes.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/virtio/Kconfig | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/drivers/virtio/Kconfig b/drivers/virtio/Kconfig
index 2e3dd3bad0..e800720657 100644
--- a/drivers/virtio/Kconfig
+++ b/drivers/virtio/Kconfig
@@ -60,9 +60,11 @@ config VIRTIO_BLK
 	  QEMU based targets.
 
 config VIRTIO_RNG
-       bool "virtio rng driver"
-       depends on VIRTIO
-       help
-         This is the virtual random number generator driver. It can be used
-	 with Qemu based targets.
+	bool "virtio rng driver"
+	depends on DM_RNG
+	depends on VIRTIO
+	default y
+	help
+	  This is the virtual random number generator driver. It can be used
+	  with QEMU based targets.
 endmenu
-- 
2.26.2

